zone/gm_commands/resetdisc_timer.cpp

```diff
@@ -2,22 +2,59 @@
 
 void command_resetdisc_timer(Client *c, const Seperator *sep)
 {
-	Client *target = c->GetTarget()->CastToClient();
-	if (!c->GetTarget() || !c->GetTarget()->IsClient()) {
-		target = c;
+	int arguments = sep->argnum;
+	if (!arguments) {
+		c->Message(Chat::White, "Usage: #resetdisc_timer all - Reset all Discipline timers");
+		c->Message(Chat::White, "Usage: #resetdisc_timer [Timer ID] - Reset Discipline timer by ID");
+		return;
 	}
 
-	if (sep->IsNumber(1)) {
-		int timer_id = atoi(sep->arg[1]);
-		c->Message(Chat::White, "Reset of disc timer %i for %s", timer_id, c->GetName());
-		c->ResetDisciplineTimer(timer_id);
+	auto target = c;
+	if (c->GetTarget() && c->GetTarget()->IsClient()) {
+		target = c->GetTarget()->CastToClient();
 	}
-	else if (!strcasecmp(sep->arg[1], "all")) {
-		c->Message(Chat::White, "Reset all disc timers for %s", c->GetName());
-		c->ResetAllDisciplineTimers();
+
+	bool is_all = !strcasecmp(sep->arg[1], "all");
+
+	if (is_all) {
+		c->Message(
+			Chat::White,
+			fmt::format(
+				"Reset all Discipline timers for {}.",
+				(
```

Comment:
Been seeing this ternary in a few PR's now. Could be simplified to a helper

eg something to the extent of `Mob::` `m->DescribeTarget(Mob *t)`

Maybe for another PR

Reply:
Yeah, I was thinking that earlier that I'm using it almost everywhere. 👍

Reply:
`m->DescribeTargetLabel(Mob *t)` maybe